cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -256,3 +258,35 @@ def document_type(self, key, value):
             raise IgnoreKey("document_type")
         else:
             raise UnexpectedValue(subfield="a")
+
+
+@model.over("_created", "(^595__)|(^961__)", override=True)
```

Comment:
same for 595, does it appear in journals?

Reply:
It only appears in one journal (check [record](https://cds.cern.ch/record/790794)). But it doesn't match the regular expresion since it's value is `No report for 2014`. So if we remove it we need to ignore it for journals or ask the librarians to clean it.

Reply:
yes please remove it from here, I have found out that 595 is for EItems. In this case journals will not have any, so please ignore 595